<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Biblioteca</class>
 <widget class="QMainWindow" name="Biblioteca">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>757</x>
      <y>10</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_prestamo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Prestamo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_devolucion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Devoluci√≥n:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBox_buscar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textBuscar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>821</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_usuario">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>usuario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_clave">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>clave</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cliente">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>-------------</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBox_libro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablaPrestamo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>821</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBox_prestamo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
